website/docs/about/overview.md

```diff
@@ -6,45 +6,63 @@ navigation:
   order: 101
 ---
 
-## What is a design system?
+## What is a design system
 
 A design system is a set of guidelines, standards, assets, and processes to help organizations design, build, and deploy products rapidly and consistently.
 
 Design systems are often equated with a UI kit or component library. In reality, an effective design system spans the entire lifecycle of a product, from ideation to iteration and everything in between.
 
-## Helios architecture
+## What is Helios
 
-While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remain largely the same across systems.
+**Helios** is the design system built by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
 
-In Helios, we offer the following:
+Helios is available and recommended for all HashiCorp **product teams using Ember.js**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```

Comment:
The phrasing of this part:

_"We also support teams using alternative frameworks or libraries, **such as HashiCorp Developer**"_

to me kind of sounds like "HashiCorp Developer" itself is an alternative framework/library...

**How about changing to:**
"We also support teams using alternative frameworks or libraries, such as React, via our design assets, [tokens](/foundations/tokens), and best practice guidance."

-OR-

"We also support teams using alternative frameworks or libraries, such as the HashiCorp Developer team which uses React, via our design assets, [tokens](/foundations/tokens), and best practice guidance."

Reply:
See the related suggested edit @Dhaulagiri has now added: https://github.com/hashicorp/design-system/pull/2012#discussion_r1544481682